_posts/2021-01-14-voicemeeter-banana.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+title:  "Introduction to Voicemeeter with a Banana"
+author: flavius
+categories: [ audio ]
+image: "voicemeeter.webp"
+featured: false
+---
+
+## What is Voicemeeter Banana
+
+Voicemeter Ba na na, na na na na, sing with me! Unless you are streaming or making a youtube video as that could result in a DMCA! Joke aside, Voicemeeter Banana, what is it? Voicemeeter Banana is the second iteration of Voicemeeter, an audio mixing software that allows advanced audio mixing, which is also provided free. You can download voicemeeter by visiting their website [here](http://voicemeeter.com/).
```

Comment:
In-paragraph hyperlinks are harder to click for mobile users. Add a single sentence as an extra paragraph just below this paragraph that is a hyperlink on it's own. For an example, look at Line #63 in the following file:

https://github.com/StreamSounds/library/blob/main/_posts/2020-12-26-how-to-use-iphone-as-webcam.md

Reply:
moved it a line down and added it to a bullet point with -